src/main/java/com/google/cloud/dns/spi/v1/HttpDnsRpc.java

```diff
@@ -380,6 +382,43 @@ public Change getChangeRequest(String zoneName, String changeRequestId, Map<Opti
     return request;
   }
 
+  @Override
+  public DnsKey getDnsKey(String zoneName, String dnsKeyId, Map<Option, ?> options) {
```

Comment:
is it possible to add unit test for ` getDnsKey` ?

Reply:
yes adding

Reply:
Done